samples/Dantooine/Dantooine.Client/Server/Startup.cs

```diff
@@ -0,0 +1,132 @@
+using System;
+using System.Net;
+using System.Net.Http;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Yarp.ReverseProxy.Forwarder;
+
+namespace Dantooine.BFF.Server
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddAntiforgery(options =>
+            {
+                options.HeaderName = "X-XSRF-TOKEN";
+                options.Cookie.Name = "__Host-X-XSRF-TOKEN";
+                options.Cookie.SameSite = Microsoft.AspNetCore.Http.SameSiteMode.Strict;
+                options.Cookie.SecurePolicy = Microsoft.AspNetCore.Http.CookieSecurePolicy.Always;
+            });
+
+            services.AddHttpClient();
+            services.AddOptions();
+
+            var openIDConnectSettings = Configuration.GetSection("OpenIDConnectSettings");
```

Comment:
Consider simply hardcoding the values instead of using `appsettings.json`.

Reply:
hardcoded